RULES/hdl/questa.go

```diff
@@ -43,7 +43,7 @@ func (rule SimulationQuesta) Build(ctx core.Context) {
 
 	srcs = append(srcs, rule.Srcs...)
 	ins = append(ins, rule.Srcs...)
```

Comment:
Add local source files after IP dependencies, i.e. Put these after the FlattenIpGraph loop?

Reply:
This does not seem to matter, but that's indeed more logically accurate. Thanks for spotting!